recipes/tpp/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "TPP" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "caed7a664be42fb9a168df699204816672e640569179cddaa42cf178d339534e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/sashimi/Trans-Proteomic%20Pipeline%20%28TPP%29/TPP%20v5.0%20%28Typhoon%29%20rev%200/TPP_5.0.0-src.tgz
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - tpp5.0.0.patch
+
+build:
+    number: 0
+    skip: true  # [osx]
+    skip: true  # [win]
```

Comment:
Bioconda is not supoprting windows builds, so this can be skiped. If you want to skip two things, the correct way would be `# [osx or win]`.